server/routes/shared/feedbackAnswersPresenter.test.ts

```diff
@@ -0,0 +1,233 @@
+import actionPlanAppointmentFactory from '../../../testutils/factories/actionPlanAppointment'
```

Comment:
I think that it would be good for this commit to make it clear that this is not new code, but rather is just being moved from `PostSessionFeedbackCheckAnswersPresenter`. Imagine a person in the future who doesn't understand a particular part of this file and so runs a `git blame` to find out more about the commit that introduced it. To them, it'll look like this code is all new, hiding all of the changes that were made to this code in its old location.

I'd say that it'd be best for this commit to, instead of introducing a new class that's a copy and paste of an existing one, also start _using_ the new class and remove the old one - i.e. incorporate the final commit `Use feedbackAnswersPresenter in checkAnswersPresenter` into this one.

Although small, focused commits are a good thing, I think that if we have to prioritise between the commits containing a self-contained change and the commits being very small, then the former probably wins. We review the commits once (where small commits are useful) but we'll probably revisit them without any context multiple times in the future (where self-contained commits are useful).

Reply:
That's good feedback, I hadn't really thought about the future value of these commits. I felt it made reviewing easier as it split it into smaller chunks, but you're right - it wouldn't be all that helpful to a future `git blame`r. Have squashed the last 3 commits into one and updated the commit message.